srtcore/core.h

```diff
@@ -1189,8 +1189,8 @@ class CUDT
     static const size_t MAX_SID_LENGTH = 512;
 
 private: // Timers functions
-    time_point m_tsTmpActiveSince; // time since temporary activated, or 0 if not temporary activated
-    time_point m_tsUnstableSince;  // time since unexpected ACK delay experienced, or 0 if link seems healthy
+    time_point m_tsFreshActivation; // time of fresh activation of the link, or 0 if not in activation phase or idle
```

Comment:
As I remember the meaning of this field, 0 is in either idle or PAST the fresh activation phase (I think it would make it if you replace "not in" with "past").